docs/installation/openyurt-prepare.md

```diff
@@ -229,5 +229,84 @@ kubectl scale --replicas=0 deployment/coredns -n kube-system
 
 ## 3. KubeProxy调整
 
-TODO
+kubeadm部署的k8s集群会为KubeProxy生成kubeconfig配置，在不配置[`Service Topology`](https://kubernetes.io/docs/concepts/services-networking/service-topology/) 和 [`Topology Aware Hints`](https://kubernetes.io/docs/concepts/services-networking/topology-aware-hints/) 情况下，KubeProxy使用这个kubeconfig拿到的endpoints是全量的。
+
+云边端场景下，边缘节点间很有可能无法互通，因此需要endpoints基于nodepool进行拓扑。直接将kube-proxy的kubeconfig配置删除，将apiserver请求经过yurthub即可解决服务拓扑问题。
+
+### KubeProxy支持流量拓扑
+
+```shell
+kubectl edit cm -n kube-system kube-proxy
+```
+
+注释掉`config.conf`文件下的`clientConnection.kubeconfig`，修改完后效果如下：
+
+```yaml
+apiVersion: v1
+data:
+  config.conf: |-
+    apiVersion: kubeproxy.config.k8s.io/v1alpha1
+    bindAddress: 0.0.0.0
+    bindAddressHardFail: false
+    clientConnection:
+      acceptContentTypes: ""
+      burst: 0
+      contentType: ""
+      #kubeconfig: /var/lib/kube-proxy/kubeconfig.conf
+      qps: 0
+    clusterCIDR: 100.64.0.0/10
+    configSyncPeriod: 0s
+// 省略
+```
+
```

Comment:
maybe we need to add more contents about that kube-proxy pod should be restarted.